### ISSUE TITLE
VGG16.py does not work with Keras 2.02 (https://github.com/fchollet/keras/blob/master/keras/applications/vgg16.py)

### ISSUE THREAD

**Issue by ptisseur** [fchollet/keras#6145]
New version of Keras does not allow to use "state of art" Keras script like VGG16.py
With Keras 2.02 there are some warning and also error trying to execute VGG16.py 
(maxpooling error).

Other issues,

Using model.layers.pop() to remove the last layer work properly. But when i try to add some other layers following the tutorial in  https://keras.io/  (using model2 = Model(input=model.input, output=predictions  ...) the command model2.summary() does not show any new layers and the model.fit()seems to work not properly which let me think that the command summary() really show that the new model2 does not receive the extra layers.

Here is the code, i whish to know what 's wrong in that code: (There is no errors)
#----------------------------------------------
model = VGG16(weights='imagenet')
#-----------------------------------------------------------
model.summary()  
#---------------------------------------------------------------  
 for layer in model.layers:
    layer.trainable = False
# load the weights
#model.load_weights('vgg16_weights.h5')

# pop last layer, insert my own
model.layers.pop()
model.layers.pop()
model.layers.pop()
#-----------------------------------------------------------
model.summary()  # 
#---------------------------------------------------------------  
 #model.add(Flatten())# flatten is still in model since we remove 3 layers (flatten is a problem....)
x=Dense(64, activation='relu')(model.output)
x=Dropout(0.5)(x)
predictions=Dense(8, activation='softmax')(x)
model2 = Model(input=model.input, output=predictions)
#-----------------------------------------------------------
model2.summary()  # == model.summary()
#---------------------------------------------------------------  

For all of that i was thinking of using Keras vs tensorflow but i am thinking return to tensorflow because debugging is much more tractable.

Thank you!

- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [x] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [ ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [x] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by fchollet** [comment]
VGG16 is not a `Sequential` model. None of what you describe is valid. To add/remove layers, see the fine-tuning example at https://keras.io/applications/

**Comment by ptisseur** [comment]
Thanks for the reply.
If model =VGG16 I have found that  x=Dense(512, activation='relu')(model.get_layer('flatten').output)  works but 
x=Flatten()(model.get_layer('block5_pool').output) does not work and i a dont understand why.
